package.json

```diff
@@ -50,6 +50,9 @@
   ],
   "devDependencies": {
     "@size-limit/preset-small-lib": "^7.0.8",
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.3.0",
+    "@types/fhir": "0.0.35",
```

Comment:
Thoughts on moving to https://github.com/fhir-typescript/fhir-typescript instead? That way we can install it as a true `dependency` not a dev dependency, and just replace all uses of `fhir4.*` with `fhirJson.*` (and an `import { fhirJson } from "@fhir-typescript/r4-core"`)

Might as well get ahead of the story when we have the option.